resources/qml/dialogs/RoomDirectory.qml

```diff
@@ -16,10 +16,10 @@ ApplicationWindow {
     id: roomDirectoryWindow
 
     visible: true
-    minimumWidth: 340
-    minimumHeight: 340
-    height: 420
-    width: 650
+    minimumHeight 320 
+    minimumWidth: 460
```

Comment:
This makes the window too big for some phone screens, afaik.

Reply:
Okay for 720*1280. Even for 480p.

Reply:
The PinePhone uses a rotated 720p screen with 2x display scaling or so. That tends to cause issues :3